src/main/java/Duke.java

```diff
@@ -1,10 +1,124 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+
+    private static final String DIVIDER = "    ____________________________________________________________";
+    private static List<Task> store = new ArrayList<Task>();
```

Comment:
```suggestion
    private static List<Task> stores = new ArrayList<Task>();
```

Reply:
Should this be written as a plural form since stores is a collection of Task objects?

Reply:
i feel that stores wouldn't really make sense here, but i do agree with changing it to something plural. Perhaps tasks would work